src/index.ts

```diff
@@ -707,6 +707,7 @@ const getAllInputFilesUnderReadme = (readMePath: string): asyncIt.AsyncIterableE
     yield* tscommonFs
       .recursiveReaddir(dir)
       .filter(filePath => path.extname(filePath) === '.json')
+      .filter(filePath => !filePath.includes('/cadl/examples/'))
```

Comment:
Avocado already has excluded files as input agruments. We don't need to hard code.

Reply:
I will also add that the exact structure of the cadl folders is still being finalized so we should be careful about backing in things like this currently.